graveyard.json

```diff
@@ -2,7 +2,7 @@
   {
     "dateClose": "2022-01-05",
     "dateOpen": "2015-05-28",
-    "description": "An Android-based embedded operating system (originally named Brillo) aimed to run on Internet of Things (IoT) devices",
+    "description": "Android Things was an Android-based embedded operating system (originally named Brillo) aimed to run on Internet of Things (IoT) devices",
```

Comment:
This sound better?
```suggestion
    "description": "Things was an Android-based embedded operating system (originally named Brillo) aimed to run on Internet of Things (IoT) devices",
```
also note:
![551124F1-B53D-4553-81D8-BAE2326AEFBB](https://user-images.githubusercontent.com/42586271/105512043-889ae800-5cc8-11eb-9568-e0de35b06ffd.jpeg)

Reply:
Oh I see what you mean now, `Android Things` would probably be best then.